dandiapi/api/management/commands/create_dev_dandiset.py

```diff
@@ -26,16 +32,19 @@ def create_dev_dandiset(*, name: str, email: str):
         user=owner, embargo=False, version_name=name, version_metadata=version_metadata
     )
 
-    uploaded_file = SimpleUploadedFile(name='foo/bar.txt', content=b'A' * 20)
+    file_size = 20
+    file_content = b'A' * file_size
+    uploaded_file = SimpleUploadedFile(name='foo/bar.txt', content=file_content)
     etag = '76d36e98f312e98ff908c8c82c8dd623-0'
+
     try:
         asset_blob = AssetBlob.objects.get(etag=etag)
     except AssetBlob.DoesNotExist:
+        # Since the SimpleUploadedFile is non-zarr asset, validation fails
+        # without a sha2_256 initially provided.
+        sha256_hash = hashlib.sha256(file_content).hexdigest()
```

Comment:
I am not quite familiar with this piece here, but we can proceed with merging and revisit once we get feedback from other members of the DANDI team in https://github.com/dandi/dandi-archive/pull/1809.